main.py

```diff
@@ -525,8 +525,6 @@ async def on_message(message):
 
 async def gather():
     gather_list_string = " ".join([member.mention for member in _okib_members])
-    # TODO combine these? can't combine the message sends, but can combine the ensure_display
-    # you doing it wront => the purpose of making a encapsulating function is to actually ensuredisplay the whole thing not everything inside it ^^
```

Comment:
@Archimonde666 I see where I went wrong now

Reply:
This is quite a nice way to code I guess :P